docs/oas_2_docgen_ascii.txt

```diff
@@ -0,0 +1,78 @@
+Load the following rows into asciiflow.com to edit:
```

Comment:
I'm concerned this could go stale. I wonder whether we could automate the generation of this file?

Reply:
Really good point. It's basically guaranteed to go stale at some point.

Reply:
Well, that's true with any documentation. It's up to the developer to keep it updated